_data/users.yml

```diff
@@ -373,6 +409,7 @@
   urltext:
   url: https://goraft.tech
 - name: Rapido
+  highlight: true
```

Comment:
I think JioSaavn or Myntra might be better to highlight over Rapido - all three are comparably successful and India-based, but we've already got a couple of other transportation companies on this list with DiDi and Lyft.

Reply:
Which one? 

Also can you do a PR for udpating the rapido logo please

Reply:
Changed to stripe after discussing on slack